ci/build/release-pipeline.yaml

```diff
@@ -155,7 +155,7 @@ spec:
               build_args:
                 component: proxy-node-gateway
               tag_file: src/.git/short_ref
-              tag_as_latest: true
+              tag_as_latest: false
```

Comment:
That is the default. I think you could safely take the line out.

It's intention is to have both, custom tag but also tag as latest by the way. I find it especially confusing, as the actual Notarising happens on docker push, meaning both tags should have been appropriately signed.

My only guess, is that the tag is part of the metadata, and the signature breaks on the initial tag? 🤔 Confusing.